### DISCUSSION TITLE
How to secure the connection between java agents and SigNoz server?

### DISCUSSION THREAD

**Discussion by uliss3s** [SigNoz/signoz#3326] [Q&A]
I'm testing SigNoz server using the docker compose setup with opentelemetry java agents configured in the tomcat server.

How can i secure this connection?

**Comment by srikanthccv** [comment]
This is old but still relevant https://medium.com/opentelemetry/securing-your-opentelemetry-collector-1a4f9fa5bd6f. If you just want to add TLS, I have added an example https://github.com/srikanthccv/otel-tls-example
